src/main/java/trackitnus/model/Model.java

```diff
@@ -230,6 +231,44 @@
      */
     ObservableList<Lesson> getFilteredLessonList();
 
+    //--------------------------------START of V1.3's new functions--------------------------------
+
+    /**
+     * @return the list of all lessons that will take place on and up to a week after the current day
+     */
+    ObservableList<Lesson> getUpcomingLessons();
+
+    /**
+     * @param date The date to query
+     * @return all lessons happens on that date
+     */
+    ObservableList<Lesson> getDayUpcomingLessons(LocalDate date);
+
+    /**
+     * @param code The module code to query
+     * @return the list of lesson for a specific module
+     */
+    ObservableList<Lesson> getModuleLessons(Code code);
+
+    /**
+     * @param code The module code to query
+     * @return the list of task for a specific module
+     */
+    ObservableList<Task> getModuleTasks(Code code);
+
+    /**
+     * @return the list of all tasks that take place on and after the current day
+     */
+    ObservableList<Task> getUpcomingTasks();
+
+    /**
+     * @param date The date to query
+     * @return the list of all tasks that take place on that specific day
+     */
+    ObservableList<Task> getDayUpcomingTasks(LocalDate date);
```

Comment:
I think change to `@return the list of all tasks that take place on the specified date`

Reply:
Okay